### PR TITLE
[jetbrains-projector] - New Chart

### PR DESCRIPTION
**Pull request by DirtyCajunRice**
**Description of the change**

**Benefits**

**Possible drawbacks**

**Applicable issues**

- fixes #

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [x] Variables are documented in the README.md (this can be done with using our helm-docs wrapper `./hack/gen-helm-docs.sh stable <chart>`)

### CODE REVIEW

File: charts/stable/jetbrains-projector/README_CHANGELOG.md.gotmpl [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+- N/A
+
+#### Changed
+
+- N/A
+
+#### Removed
+
+- N/A
```
Comment (bjw-s) [inline charts/stable/jetbrains-projector/README_CHANGELOG.md.gotmpl:24]:
```suggestion
#### Added

- Initial version
```

File: charts/stable/jetbrains-projector/values.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- See list of available images [here](https://github.com/JetBrains/projector-docker)
+  registry: registry.jetbrains.team/p/prj/containers
+  repository: projector-pycharm-c
+  pullPolicy: IfNotPresent
```
Comment (bjw-s) [inline charts/stable/jetbrains-projector/values.yaml:12]: Would it make sense to default this to `Always` since there is a `latest` tag set by default?

**Review by bjw-s — CHANGES_REQUESTED** [review]

File: charts/stable/jetbrains-projector/values.yaml (hunk 2 above)
Comment (DirtyCajunRice) [reply charts/stable/jetbrains-projector/values.yaml:14]: Containers have been pushed so this is fixed with proper versioning

**Review by onedr0p — APPROVED** [review]

**Comment by onedr0p** [comment]
@all-contributors add @DirtyCajunRice for code

**Pull request merged**